source/extensions/filters/http/grpc_http1_reverse_bridge/filter.cc

```diff
@@ -92,6 +98,19 @@ Http::FilterDataStatus Filter::decodeData(Buffer::Instance& buffer, bool) {
 
 Http::FilterHeadersStatus Filter::encodeHeaders(Http::HeaderMap& headers, bool) {
   if (enabled_) {
+    if (headers.ContentType() == nullptr) {
```

Comment:
I'd probably move the `auto content_type = ...` to before this if block instead of calling `headers.ContentType()` twice (it's a constant time lookup, but I think it reads a bit clearer)